_includes/payment-order-other-features.md

```diff
@@ -296,9 +296,20 @@ The `view-paymentorder` operation contains the URI of the JavaScript that needs
 
 ### Update Order
 
-Change amount and vat amount on a payment order. If you implement `UpdateOrder`
-**you need to `refresh()`** the [Payment Menu front end][payment-menu-front-end] so the
-new amount is shown to the end customer.
+`Updateorder` is used when there is a change in the amount, vat amount or
+added/removed order items in a payment order. 
+
+{% include alert.html type="informative" icon="info" body="If you implement 
+`UpdateOrder` you need to `refresh()` the 
+[Payment Menu front end][payment-menu-front-end]
```

Comment:
Does reference links work with `include` or does this have to be converted to an inline link?

Reply:
I dont think we need this link at all